packages/cmrjs/cmr-utils.js

```diff
@@ -54,6 +54,22 @@ function isCMRFile(fileobject) {
   return isCMRFilename(cmrfilename);
 }
 
+/**
+ * Reduce granule object array to CMR files array
+ *
+ * @param {Object} granules = granule objects array
+ *
+ * @returns {Array<Object>} - CMR file object array: { filename, granuleId }
+ */
+function reduceGranulesToCmrFileObjects(granules) {
```

Comment:
Consider removing the `reduce` from the function name?  `granulesToCmrFileObjects` or `cmrFileObjectsFromGranules`.

Reply:
I used `reduce...` to make it clear that this is a reducing operation, vs a `map` or `forEach`. I'm open to hearing why it should change, though.

Reply:
It's not really a reducing operation, since you're going from an array to an array.  (You're not taking a list of values and reducing them to a single value.)  This is more of a data transformation: given a list of granules, give me the list of files.  As for "it's not a map", it kind of is, with a built-in filter operation.  It's not a hill I'm going to die on, but I think that a function name of "x to y" or "y from x" is a better description.